### PR TITLE
Adding showIf support in InputField and ConfigValue

### PR DESCRIPTION
**Pull request by pyros82**
This goes with the 0.5.7 update of chart-ext and 0.5.1 of the sdk+backend - the frontend will now respect the showIf field of UiSchemaInput, giving it priority over showIfAll (as the backend does).

I was able to test this quite thinly - if you have a way to verify this too, please do.

Thanks

### CODE REVIEW

File: src/components/inputs/InputField.vue [hunk 1]
```diff
@@ -124,12 +125,24 @@ export default defineComponent({
         );
 
         const visible = computed(
-            () =>
-                !props.input.showIfAll ||
-                props.input.showIfAll.every(
+            () => {
+                if (props.input.showIf != undefined) {
```
Comment (popen2) [inline src/components/inputs/InputField.vue:129]: I think testing for undefined is too strict (and in any case should be done with `!==` to ensure it's actually `undefined` and not `null`). If the value is `null` we don't want to check it either, and JsonLogic has to be an object, so it feels fine.
```suggestion
                if (props.input.showIf) {
```

File: src/components/values/ConfigValue.vue [hunk 2]
```diff
@@ -96,13 +97,28 @@ export default defineComponent({
...
-                props.input &&
-                (!props.input.showIfAll ||
-                    props.input.showIfAll.every(
-                        (fv) => props.allValues[fv.field] === fv.value
-                    ))
-        );
+            () => {
+                if (!props.input) { return false }
+
+                if (props.input.showIf != undefined) {
```
Comment (popen2) [inline src/components/values/ConfigValue.vue:103]: ditto
```suggestion
                if (props.input.showIf) {
```

File: src/components/values/ConfigValue.vue [hunk 3]
```diff
@@ -96,13 +97,28 @@ export default defineComponent({
...
+                } else if (props.input.showIfAll != undefined) {
+                    return props.input.showIfAll.every(
+                    (fv) => props.allValues[fv.field] === fv.value
+                )
+                } else {
+                    return true
+                }
+            }
+
+            );
```
Comment (popen2) [inline src/components/values/ConfigValue.vue:121]:
```suggestion
        );
```

**Review by popen2 — APPROVED** [review]
Some minor comments, but looks good!

**Comment by popen2** [comment]
Thank you @pyros82!

I'll include this with two more bugfixes in 0.5.1, will release it by next week.

File: src/components/inputs/InputField.vue (hunk 1 above)
Comment (pyros82) [reply src/components/inputs/InputField.vue:129]: I inherently get the heebie-jeebies by trueish values, but I guess it's more idiomatic in js. Thanks!

**Pull request merged**